addons/calendar/models/calendar_event.py

```diff
@@ -486,6 +497,13 @@ def write(self, values):
         if 'partner_ids' in values:
             values['attendee_ids'] = self._attendees_values(values['partner_ids'])
             update_alarms = True
+            if self.videocall_channel_id:
+                new_partner_ids = []
+                for value in values['partner_ids']:
+                    op = value[0]
+                    if op in (4, 6):
```

Comment:
I've not checked the rest of the code in details but the suggestion to use variable instead of number would probably apply here.